package.json

```diff
@@ -40,6 +40,7 @@
     "commander": "^6.0.0",
     "http-proxy": "^1.18.1",
     "kleur": "^4.1.1",
+    "playwright-core": "=1.6.2",
```

Comment:
This is not correct, the examples works as node_modules are traversed from local -> parent -> root directory.

We can get rid of `playwright-core` deps totally and just not use any deps in the example. We are just importing them for making intensense work. 

Lets change this to https://github.com/elastic/synthetics/blob/73699e02c918206b9435e8af86b6ce5f5104fdd8/examples/todos/helpers.ts#L4 `import { Page } from "playwright-chromium"` if we need auto completion or we can even get rid of those and keep the example minimal.

Reply:
@vigneshshanmugam well, intellisense working is an important feature. We should discourage users in our examples from using `playwright-chromium` because that's not correct for multi-browser support (and confusing).

Additionally, one of the problems with our examples folder is the traversal to the root directory. Users will use the `todos` example in their projects, so everything in the `todos` folder should work even if it's removed from the context of our `synthetics` parent project. In fact, I've been thinking it'd be much better if we could make `todos` a sibling folder instead of a subfolder for this very reason.